py/picca/fitter2/chi2.py

```diff
@@ -195,13 +197,34 @@ def fastMC(self):
             d.ico = d.ico/s
             d.cho = cholesky(d.co)
 
+        self.fiducial_values = self.best_fit.values.copy()
+        if len(self.fidfast_mc) != 0:
```

Comment:
would simply `for p in self.fidfast_mc:` work?
If empty then do not enter the loop